### PR TITLE
Information architecture: reordering the sections

### PR DESCRIPTION
**Pull request by eviau-sat**
#### Summary
Reorder the sections so that the content that the user needs is first!

#### Motivation
This should help make the documentation more skimmable.

#### Supporting details
None for now.

#### Related issues
I did not see any.

#### Metadata

This PR…
-->
- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/getting_started_with_the_web/installing_basic_software/index.md [hunk 1]
```diff
@@ -53,6 +41,20 @@ Before going on, you should install at least two of these browsers and have them
 
 Some examples will need to be run by a web server to work successfully. You can find out how to do this in [How do you set up a local testing server?](/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server)
 
+## What tools do the professionals use?
+
+That looks like a scary list, but fortunately, you can get started in web development without knowing anything about most of these.
```
Comment (queengooborg) [inline files/en-us/learn/getting_started_with_the_web/installing_basic_software/index.md:46]:
```suggestion
The following looks like a scary list, but fortunately, you can get started in web development without knowing anything about most of these.
```

File: files/en-us/learn/getting_started_with_the_web/installing_basic_software/index.md [hunk 2]
```diff
@@ -15,21 +15,9 @@ tags:
...
-- **An automation system,** like [Webpack](https://webpack.js.org/), [Grunt](https://gruntjs.com/), or [Gulp](https://gulpjs.com/) to automatically perform repetitive tasks, such as minifying code and running tests.
-- Libraries, frameworks, etc., to speed up writing common functionality. A library tends to be an existing JavaScript or CSS file that provides ready-rolled functionality for you to make use of in your code. A framework tends to take this idea further, offering a complete system with some custom syntaxes for you to write a web app on top of.
-- More tools besides!
-
 ## What tools do I actually need, right now?
 
-That looks like a scary list, but fortunately, you can get started in web development without knowing anything about most of these. In this article, we'll just set you up with a bare minimum — a text editor and some modern web browsers.
+In this article, we'll just set you up with a bare minimum — a text editor and some modern web browsers.
```
Comment (queengooborg) [inline files/en-us/learn/getting_started_with_the_web/installing_basic_software/index.md:20]:
```suggestion
In this article, we'll set you up with a bare minimum — a text editor and some modern web browsers.
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Hey there, thank you for your PR, and welcome to MDN web docs!  I've just got a few grammatical and wording changes to suggest, but overall this is LGTM!

**Comment by estelle** [comment]
Thank you for doing this. I agree the original version was a bit scary and overwhelming.

**Review by queengooborg — APPROVED** [review]
Thank you for the quick changes, LGTM!

**Pull request merged**